code/analytics/src/phone/java/com/adobe/marketing/mobile/Analytics.java

```diff
@@ -22,7 +22,7 @@
 
 public class Analytics {
     private static final String LOG_TAG = "Analytics";
-    private static final String EXTENSION_VERSION = "2.0.0";
+    private static final String EXTENSION_VERSION = "2.0.1";
```

Comment:
Can `AnalyticsConstants.EXTENSION_VERSION` be used here instead of redefining the value?

Reply:
not without making the constants public from the internal package

Reply:
Is there a test to validate Analytics.EXTENSION_VERSION == AnaltyicsConstants.EXTENSION_VERSION == moduleVersion ?